samples/bluetooth/radio_notification_cb/sample.yaml

```diff
@@ -0,0 +1,13 @@
+sample:
+  description: Connection event callback sample
+  name: Connection event callback
+tests:
+  sample.bluetooth.conn_evt_cb:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpunet
+    platform_allow: nrf52dk/nrf52832 nrf52840dk/nrf52840 nrf5340dk/nrf5340/cpunet
```

Comment:
does not work on 54?

Reply:
Not yet. Need to add EGU definitions here: https://github.com/zephyrproject-rtos/zephyr/blob/main/modules/hal_nordic/nrfx/Kconfig#L64. I believe we can do that separately to unblock other work

Reply:
Will be possible after https://github.com/zephyrproject-rtos/zephyr/pull/75866 is in NCS